pkg/artifacts/signable_test.go

```diff
@@ -317,11 +319,52 @@ func TestExtractOCIImagesFromResults(t *testing.T) {
 		return a.String() < b.String()
 	})
 	if !cmp.Equal(got, want, ignore...) {
-		t.Fatalf("not the same %s", cmp.Diff(got, want, ignore...))
+		t.Fatalf("not the same %s", cmp.Diff(want, got, ignore...))
 	}
 }
 
-func digest(t *testing.T, dgst string) name.Digest {
+func TestExtractIntotoSignableTargetFromResults(t *testing.T) {
```

Comment:
It would be super to add a couple of test cases to verify things are being ignored properly given the complex logic in `extractTargetFromResults`

Reply:
Ah yeah they are covered in TestExtractIntotoSignableTargetFromResults cases (miss_target_name_INTOTO_TARGET_DIGEST, wrong_digest_format_INTOTO_TARGET_NAME, etc.). I've added a few more in TestExtractOCIImagesFromResults as well